src/Hpp/RunHpp.hs

```diff
@@ -571,16 +582,15 @@ macroExpansion :: (Monad m, HppCaps m)
 macroExpansion = do
   lift await >>= \case
     Nothing -> return Nothing
+    Just ln@(Important "#":rst) -> do
```

Comment:
I'm not sure that this change is follows the spec, but I tried out an example with a preprocessor directive with some space prepended to it and it failed, so I assume most implementations expect there to be no whitespace from the beginning of the line to the first `#` of the directive.

I encountered this when compling a module with an input like:
```
{-# LANGUAGE MagicHash
    #-}
```
The 2nd line is treated as a directive and becomes an unknown command. Maybe this change should only be enabled when the flag I mentioned earlier is turned on?

Reply:
I don't feel strongly about this, but making it configurable does seem like the right thing to do.